In [1]:
import pandas as pd
data = pd.read_csv("ForeignGifts_edu.csv")
data["Institution Name"].unique()

<StringArray>
[                       'Jacksonville State University',
                                      'Troy University',
                                'University of Alabama',
                  'University of Alabama at Birmingham',
                          'University of South Alabama',
                             'Arizona State University',
                          'Northern Arizona University',
                          'University of Arizona (The)',
                               'University of Arkansas',
                                     'Biola University',
 ...
                      'DigiPen Institute of Technology',
                           'Soka University of America',
     'Keck Graduate Institute of Applied Life Sciences',
                         'Columbia Southern University',
                     'University of California, Merced',
                   'Hult International Business School',
                             'Presidio Graduate School',
          'C

In [2]:
uva = data[data['Institution Name'] == "University of Virginia"]

UVA Gifting Statistics

In [3]:
print(f"Total amount recieved: ${uva['Foreign Gift Amount'].sum()}")
print(f"Average amount recieved: ${uva['Foreign Gift Amount'].mean()}")
print(f"Median amount recieved: ${uva['Foreign Gift Amount'].sum()}")
print(f"Number of gifts recieved: {uva['Foreign Gift Amount'].count()}")


Total amount recieved: $22189238
Average amount recieved: $226420.79591836734
Median amount recieved: $22189238
Number of gifts recieved: 98


UVA's Donors

In [4]:
uva['Country of Giftor'].value_counts()

Country of Giftor
ENGLAND            31
ISRAEL             12
BANGLADESH         10
SWEDEN             10
FINLAND             7
KOREA               6
FRANCE              5
THE NETHERLANDS     4
TANZANIA            3
GUERNSEY            3
SINGAPORE           2
SWITZERLAND         2
JAPAN               1
BERMUDA             1
RWANDA              1
Name: count, dtype: int64

In [5]:
top_giftors = uva.groupby('Country of Giftor')['Foreign Gift Amount'].sum()
top_giftors.sort_values()

Country of Giftor
FRANCE              250383
JAPAN               265970
RWANDA              284135
ISRAEL              623655
THE NETHERLANDS     725670
SINGAPORE           805000
KOREA               862787
BERMUDA             900000
FINLAND            1008384
GUERNSEY           1500000
ENGLAND            1950666
SWITZERLAND        2000000
TANZANIA           2776447
SWEDEN             4007552
BANGLADESH         4228589
Name: Foreign Gift Amount, dtype: int64

Most Gifts Donated to UVA:

    England (31)

    Israel (12)

    Bangladesh/Sweden (10)



Top Giftors by Amount:

    BANGLADESH         $4228589

    SWEDEN             $4007552

    TANZANIA           $2776447


In [6]:
pd.set_option('display.max_rows', None)
data['Institution Name'].value_counts()

Institution Name
University of California, Los Angeles                            3916
California Institute of Technology                               2502
Columbia University in the City of New York                      1127
Ohio State University (The)                                      1014
Johns Hopkins University                                          899
University of Michigan - Ann Arbor                                715
Stanford University                                               668
Cornell University                                                653
University of Pennsylvania                                        625
Massachusetts Institute of Technology                             582
Yale University                                                   550
Harvard University                                                453
University of Colorado Denver                                     436
University of Chicago (The)                                       401
Ari

In [7]:
pd.reset_option('display.max_rows')
acc = data[data['Institution Name'].isin(["University of Virginia", 
           "University of North Carolina - Chapel Hill",
           'North Carolina State University',
           'Duke University',
           'University of Miami',
           'Clemson University',
           'Georgia Institute of Technology'
])]
acc["Institution Name"].value_counts()

Institution Name
University of North Carolina - Chapel Hill    286
Georgia Institute of Technology               203
Duke University                               145
University of Virginia                         98
Clemson University                             94
University of Miami                            27
North Carolina State University                14
Name: count, dtype: int64

In [8]:
acc.groupby('Institution Name')['Foreign Gift Amount'].sum().sort_values()

Institution Name
Clemson University                              4912287
University of Miami                            18745285
University of Virginia                         22189238
North Carolina State University                23762788
University of North Carolina - Chapel Hill     41292544
Georgia Institute of Technology                77979361
Duke University                               343699498
Name: Foreign Gift Amount, dtype: int64

UVA ranks third to last when in this sample of other ACC universities sorted by dollar amount of gifts recieved.

In [9]:
acc.groupby('Institution Name')['Foreign Gift Amount'].mean().sort_values()

Institution Name
Clemson University                            5.225837e+04
University of North Carolina - Chapel Hill    1.443795e+05
University of Virginia                        2.264208e+05
Georgia Institute of Technology               3.841348e+05
University of Miami                           6.942698e+05
North Carolina State University               1.697342e+06
Duke University                               2.370341e+06
Name: Foreign Gift Amount, dtype: float64

UVA ranks third to last in this sample group when sorted by average gift size.

In [10]:
pd.set_option('display.max_rows', None)
acc.groupby('Institution Name')['Giftor Name'].value_counts()


Institution Name                            Giftor Name                        
Clemson University                          BMW                                     43
                                            MHI Vestas Offshore Wind                13
                                            Kuwait Embassy                          10
                                            Honda R&D Americas, Inc.                 9
                                            Kuwait Cultural Office-Los Angeles       7
                                            BMW Mfg. Corp. LLC                       7
                                            Samsung                                  3
                                            Honda Motor Company                      2
Duke University                             University of Nazarbayev                 6
                                            Swedish Orphan Biovitrum AB              5
                                            Sanofi

It appears that most most of the universities with strong histories of receiving gifts have more diverse lists of giftors over having one donor that has given gifts frequently. Because of this, UVA should try to expand its network of partner companies and foster relationships with foreign universities. A potential way of doing this is offering collaborative capstone projects with industry partners in more undergraduate and graduate schools. UVA could also discuss implementing more exchange programs with universities abroad to build a stronger network  of international institutions.

In [13]:
acc['Giftor Name'].value_counts()

Giftor Name
Shire Human Genetics Therapies, Inc    107
BMW                                     43
CIDRZ                                   35
GSK                                     31
CAF American Donor Fund                 14
T-BASE COMMUNICATIONS INC               13
MHI Vestas Offshore Wind                13
Novo Nordisk                            10
Intl Agency for Research on Cancer      10
Kuwait Embassy                          10
T-BASE COMMUNICATIONS INC/OTTAWA         9
China Medical University                 9
Honda R&D Americas, Inc.                 9
Kirkhouse Trust                          9
argenx BVBA                              8
MSD                                      7
GW Research Ltd                          7
BMW Mfg. Corp. LLC                       7
Kuwait Cultural Office-Los Angeles       7
IHI Corporation                          6
KING ABDULLAH UNIV OF SCI & TECH (K      6
SAMSUNG ELECTRONICS                      6
S.F. EXPRESS                             6

In [14]:
acc['Country of Giftor'].value_counts()

Country of Giftor
CANADA                  159
ENGLAND                 141
GERMANY                  58
FRANCE                   44
KOREA                    39
JAPAN                    38
ZAMBIA                   36
SWITZERLAND              31
SAUDI ARABIA             30
DENMARK                  28
SWEDEN                   26
CHINA                    24
BELGIUM                  19
KUWAIT                   17
SOUTH KOREA              16
TAIWAN                   15
THE NETHERLANDS          14
KAZAKHSTAN               13
ISRAEL                   12
HONG KONG                10
BANGLADESH               10
IRELAND                   9
FINLAND                   9
INDIA                     6
AUSTRALIA                 5
TANZANIA                  5
SINGAPORE                 5
COLOMBIA                  4
PANAMA                    4
SWAZILAND                 3
CAYMAN ISLANDS (THE)      3
QATAR                     3
SPAIN                     3
PERU                      3
SOUTH AFRICA              3
GU

In [15]:
uva['Giftor Name'].value_counts()

Giftor Name
Kirkhouse Trust                        9
AutoLiv Development AB                 5
University of Newcastle                4
Danisco Sweeteners Oy                  4
InSightec-Image Guide Treatment Ltd    3
Charities Aid Foundation               3
Intl. Cntr for Diarrihoeal Disease     3
Intrnl Centre Diarrheal Disease        3
London School of Economics             3
FKD Therapies Oy                       3
GW Research Ltd                        3
US-Israel Binational Sci Fdn           2
AstraZeneca                            2
Hyundai Motor Company                  2
Rainbow Foundation                     2
Harvest Group Holding                  2
Intnl Centre for Diarrhoeal Disease    2
Genfit                                 2
Pantarhei Oncology BV                  2
West Yorkshire Police                  1
CureTech Ltd                           1
Branco Weiss Institute                 1
NovoCure Ltd                           1
Atox Bio Ltd.                          1
TauR

In [16]:
uva['Country of Giftor'].value_counts()

Country of Giftor
ENGLAND            31
ISRAEL             12
BANGLADESH         10
SWEDEN             10
FINLAND             7
KOREA               6
FRANCE              5
THE NETHERLANDS     4
TANZANIA            3
GUERNSEY            3
SINGAPORE           2
SWITZERLAND         2
JAPAN               1
BERMUDA             1
RWANDA              1
Name: count, dtype: int64